cspyce/cspyce1.py

```diff
@@ -106,9 +108,12 @@
 import numpy as np
 import textwrap
 
-import cspyce.cspyce0 as cspyce0
+# import cspyce.cspyce0 as cspyce0
```

Comment:
Do we need these commented-out lines?

Reply:
I made no changes to cspyce1.py.  
But I deleted this